eng/pipelines/templates/steps/get-tagged-code.yml

```diff
@@ -0,0 +1,40 @@
+parameters:
+  - name: DestinationDirectory
+    type: string
+    default: ''
+  - name: Package
+    type: string
+    default: ''
+  - name: CheckoutPath
+    type: string
+    default: ''
+  - name: Version
+    type: string
+    default: ''
+          
+steps:
+    # TODO, re-use eng/common sparse-checkout.yml
```

Comment:
This is going to be ripped out and replaced with a python script. This will enable us to easily run this locally.

Reply:
Is there a reason you cannot use the sparse-checkout template?

Reply:
Yes. `checkout: none` is in the lead. You can only do this a single time throughout the whole job template.

I have attempted to add a condition manually to the template, still no dice. Going to go with this until I replace it with the python script part (for easy local runs of this thing).

Reply:
FYI @benbp we might need to think about some flexibility around checkout:none so that we could do multiple sparse checkouts in the same pipeline.

Reply:
@scbedd and I talked about a parameter to make `sparse-checkout.yml` functional when there are multiple uses in the same job (basically an optional skip for `checkout: none`.

Reply:
yar! I submitted a PR to azure-sdk-tools that I need to get merged before this.